lib/diagnostics_channel.js

```diff
@@ -136,10 +139,92 @@ function hasSubscribers(name) {
   return channel.hasSubscribers;
 }
 
+function bindStore(name, store, build = (v) => v) {
```

Comment:
`diagnosticChannel.bindStore` reads to me that the diagnostic channel can only be bound to one single AsyncLocalStorage at one time. I believe this is not the intention, right?

Reply:
No, the intent is the reverse, that the store is being bound to the storage channel.